components/clarinet-deployments/src/lib.rs

```diff
@@ -120,7 +121,6 @@ pub fn update_session_with_genesis_accounts(
                 session.set_tx_sender(wallet.address.to_address());
             }
         }
-        session.load_boot_contracts();
```

Comment:
`update_session_with_genesis_accounts()` should not be responsible for calling laod_boot_contracts.

It doesn't have any impacts to remove it because every `update_session_with_genesis_accounts` calls are also followed by `update_session_with_contracts_executions` which also loads boot contracts